resources/typemappings/Genome.yaml

```diff
@@ -0,0 +1,27 @@
+storage-type: WS
+types:
+    KBaseGenomes.Genome:
+        default:
+           -
+               type: Genome
```

Comment:
Hmm, if most new genomes are going to be new style, wouldn't it be better to have the default be v2?

Reply:
With the current style, wouldn't I have to list every revision from 1-12 to point them to the old style? This is much more concise and any new revisions to genome would have to be reflected here anyway.

Reply:
yes, but if 1-12 are listed that means if you make more changes to the genome you don't need to update the mappings unless the changes are backwards incompatible.

Reply:
Currently if you make even a trivial change to the the ws spec it'll get indexed as the v1 spec and fail indexing.